src/transformers/modeling_utils.py

```diff
@@ -158,7 +158,12 @@ def convert_file_size_to_int(size: Union[int, str]):
     if size.upper().endswith("KIB"):
         return int(size[:-3]) * (2**10)
     if size.upper().endswith("GB"):
-        return int(size[:-2]) * (10**9)
+        if size.endswith("Gb"):
+            # gigabits
+            return int(size[:-2]) * (10**9 >> 3)
```

Comment:
Not sure I understand this. This should be `// 8`  as 1 gigabyte = 8 gigabits. Also if we start supporting this, it should be the same for kilos and megas.

Reply:
> Not sure I understand this. This should be `// 8` as 1 gigabyte = 8 gigabits.

Yes, `>> 3` does the same as `// 8`.

> Also if we start supporting this, it should be the same for kilos and megas.

IMO it makes sense to be as precise as possible.

P.S. I did the same for kilos and megas.